MembershipChecker.js

```diff
@@ -86,6 +94,9 @@ function getLibOptions(){
   return AdminPanel.getPanelValues("MembershipChecker");
 }
 
+function wasLastJoined(passed_bool){
+  User.setProperty(LIB_PREFIX + "wasLastJoined", User.getProperty(LIB_PREFIX + "wasLastJoined", []).push(passed_bool), "json");
```

Comment:
you use prop here. It can be possible without this prop: we can use already exists prop userData with chats.

So this prop looks like extra not needed prop

Reply:
Yes, I have done it in [This Pull](https://github.com/bots-business/store-libs/pull/22)